htdocs/societe/class/societe.class.php

```diff
@@ -1936,6 +1947,12 @@ function add_commercial(User $user, $commid)
 	 */
 	function del_commercial(User $user, $commid)
 	{
+		$error=0;
+		$this->commercial_modified = $commid;
+				
+		$result=$this->call_trigger('COMPANY_DEL_COMMERCIAL',$user);
```

Comment:
Can you name your trigger 'COMPANY_UNLINK' to match naming rules used by other objects